sources/packages/web/src/components/common/ProgramOfferingDetailHeader.vue

```diff
@@ -89,7 +86,13 @@ export default {
   setup(props: any) {
     const router = useRouter();
     const { dateOnlyLongString } = useFormatters();
-
+    const showApprovalDetails = computed(
+      () =>
+        props.headerDetails.assessedBy &&
+        props.headerDetails.assessedDate &&
+        (props.headerDetails.status !== ProgramStatus.Pending ||
```

Comment:
Is the status not enough to define if we should show or not the assessment details?
Is there a scenario where it will be different than pending and do not have the `assessedBy` and `assessedDate`.

Reply:
While re-editing the program the UI and if it is set to pending then UI gets disturbed again.